### DISCUSSION TITLE
OB examples

### DISCUSSION THREAD

**Discussion by jpdlpassos** [Open-Orange-Button/Orange-Button-Taxonomy#233] [Q&A]
Hi,

Is is possible to get anonymized versions of actual solar plants in OB format ?
It would be helpful to clarify some doubts we have while trying to adopt this format.

Thank you

**Comment by cwhanse** [comment]
@jpdlpassos I hoped someone else would respond, and then forgot you had asked. I'm not aware of any publicly-shared instances of system descriptions in OB.
